src/output-json-tls.c

```diff
@@ -333,16 +338,16 @@ static void JsonTlsLogClientCert(
         SCJsonBuilder *js, SSLStateConnp *connp, const bool log_cert, const bool log_chain)
 {
     if (connp->cert0_subject != NULL) {
-        SCJbSetString(js, "subject", connp->cert0_subject);
+        SCJbSetStringFromBytes(js, "subject", connp->cert0_subject, connp->cert0_subject_len);
```

Comment:
this leads to a behavior change. Previously we'd add `"subject":""` if there was no (or zero len) value. Now we omit it. Caught this in a private test failure.

I *think* omitting it may be more idiomatic, right @jasonish ?